src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java

```diff
@@ -25,6 +40,10 @@
     static final String currentUserDirectory = "CurrentUser";
     static final String myCertificateStore = "My";
 
+    // FOR NON WINDOWS MACHINES
+    String keyStorePath = null;

```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.design.VisibilityModifierCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java L44

Variable &apos;keyStorePath&apos; must be private and have accessor methods.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1938480846>*